modules/ROOT/pages/api-gateway-encryption-mule4.adoc

```diff
@@ -47,7 +47,7 @@ This command outputs your encrypted client ID:
 > smzbg+aV2HbiC9Fv8KBsZpD5PlXiplsfhZ42RWNEbHhUeZ9ZIDftljUx6sYpuQ+b
 --
 
-== Passing your Encrypted Client ID and Secret
+== Passing your Encrypted Client ID and Client Secret
```

Comment:
```suggestion
== Passing Your Encrypted Client ID and Client Secret
```

Reply:
What do you mean by "passing," and to what target? Is "passing" the correct term? It has multiple meanings.

Reply:
Passing an argument or parameter is a common term in programming.